src/RequestGroupMan.cc

```diff
@@ -105,6 +105,9 @@ RequestGroupMan::RequestGroupMan(
     int maxSimultaneousDownloads, const Option* option)
     : maxSimultaneousDownloads_(maxSimultaneousDownloads),
       numActive_(0),
+      optimizeSimultaneousDownloads_(false),
+      optimizeSimultaneousDownloadsCoeffA_(5.),
+      optimizeSimultaneousDownloadsCoeffB_(25.),
```

Comment:
The above 3 lines should be put before `numActive_(0),`.   Compiler shows warnings like `field 'numActive_' will be initialized after field 'optimizeSimultaneousDownloads_' [-Werror,-Wreorder]`